dspace/config/dspace.cfg

```diff
@@ -1585,6 +1592,13 @@ request.item.type = all
 request.item.helpdesk.override = false
 
 
+#---------------------------------------------------------------#
+#----------ITEM DATABASE STATUS SCRIPT CONFIGURATION------------#
```

Comment:
We should change this to say "SOLR DATABASE RESYNC SCRIPT CONFIGURATION" (or similar)